web/static/js/actions/questionnaire.js

```diff
@@ -0,0 +1,125 @@
+import * as api from '../api'
+import { buildNewStep } from '../reducers/questionnaire'
+
+export const NEW = 'QUESTIONNAIRE_NEW'
+export const FETCH = 'QUESTIONNAIRE_FETCH'
+export const RECEIVE = 'QUESTIONNAIRE_RECEIVE'
+export const CHANGE_NAME = 'QUESTIONNAIRE_CHANGE_NAME'
+export const TOGGLE_MODE = 'QUESTIONNAIRE_TOGGLE_MODE'
+export const ADD_STEP = 'QUESTIONNAIRE_ADD_STEP'
+export const DELETE_STEP = 'QUESTIONNAIRE_DELETE_STEP'
+export const CHANGE_STEP_TITLE = 'QUESTIONNAIRE_CHANGE_STEP_TITLE'
+export const CHANGE_STEP_PROMPT_SMS = 'QUESTIONNAIRE_CHANGE_STEP_PROMPT_SMS'
+export const CHANGE_STEP_STORE = 'QUESTIONNAIRE_CHANGE_STEP_STORE'
+export const ADD_CHOICE = 'QUESTIONNAIRE_EDITOR_ADD_CHOICE'
+export const DELETE_CHOICE = 'QUESTIONNAIRE_EDITOR_DELETE_CHOICE'
+export const CHANGE_CHOICE = 'QUESTIONNAIRE_EDITOR_CHANGE_CHOICE'
+
+export const fetchQuestionnaire = (projectId, questionnaireId) => (dispatch, getState) => {
```

Comment:
Ah, I see. So maybe the public API should be `fetchQuestionnaire` and the private one be `forceFetchQuestionnaire`?